docs/sources/configuration/_index.md

```diff
@@ -791,6 +791,11 @@ results_cache:
 # CLI flag: -frontend.forward-headers-list
 [forward_headers_list: <list of strings> | default = []]
 
+# The downstream querier is requested to answer in the accepted format. Can be
+# 'json' or 'protobuf'. Note: Both will still be routed over GRPC.
+# CLI flag: -frontend.accepted-query-response-format
+[accepted_query_response_format: <string> | default = "json"]
```

Comment:
We handle both based on the `Accept` header. However, this option tells the frontend to ask for the Protobuf encoded response instead of the JSON encoded response. Maybe the option is not clear enough.

Reply:
maybe the name just needs to change? `requested_query_response_format`, the comment could then be `The downstream querier is requested to send responses to the query-frontend in the following format.`